### DISCUSSION TITLE
Looking for alternative to sponge

### DISCUSSION THREAD

**Discussion by KaelaniRevyruun** [IzzelAliz/Arclight#612] [Q&A]
Hey, I'm looking at alternatives for spongeforge, cos any time I add sponge to my server it crashes after a minute or so and says "overworld settings missing", and the sponge discord seems to have not a single helpful person; I asked about the issue there and they basically just told me "oh that sounds like a mod changing the dimension type of the overworld." Problem is, they don't tell me anything besides that :/

So, with the above in mind, does arclight work for 1.16.5, and is it compatible with worldedit forge, griefdefender, nucleus, luckperms and any economy plugins? :P And relatedly, are any mods known to cause issues?

Thanks in advance for your time :D

**Comment by Rasgnarok** [comment]
"So, with the above in mind, does arclight work for 1.16.5, and is it compatible with worldedit forge, griefdefender, nucleus, luckperms and any economy plugins?"
No, Arclight isn't Sponge and therefore you can't run Sponge plugins on Arclight.

**Reply by KaelaniRevyruun** [reply]
There's a couple of different versions of each though... Like I know there's a spigot version of luckperms for example xD I'm just wondering if there's compatible versions of each of the plugins I mentioned that work with arclight, and whether I can have both mods and plugins working together on arclight; right now sponge just crashes the server with any mods added, plugins or not :/

**Comment by heipiao233** [comment]
Why don't you have a try? If these plugins and mods crashes your server, you can make an issue.
